### ISSUE TITLE
Label smoothing in the focal loss

### ISSUE THREAD

**Issue by showgood163** [Hsuxu/Loss_ToolBox-PyTorch#4]
https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/master/FocalLoss/FocalLoss.py#L75

Maybe the label smoothing is not correct here? The sum of a probability distribution should be 1.

A possible implementation:

one_hot_key = torch.clamp(one_hot_key, self.smooth / (self.num_class - 1), 1.0 - self.smooth)
